src/views/components/name_modal.vue

```diff
@@ -0,0 +1,105 @@
+<style>
+</style>
+
+<template>
+        <v-card>
+            <v-card-title>Edit Name</v-card-title>
+
+            <v-layout row wrap>
+                <v-flex xs12 sm6 md4>
```

Comment:
The sizing information should be part of the dialog that wraps it, not the widget itself.